project-koku/koku
# [Cost 2372] Add HCS market place data for AWS and Azure to dev tools

Fixes [COST-2372](https://issues.redhat.com/browse/COST-2372)

Changes proposed in this PR:
* add marketplace data to customer data in dev env
* update dev dep for Nise

Testing Instructions:
1.  setup normally by running
`make create-test-customer`
`make load-test-customer-data`
2. now using Trino cli run the following query
`trino --server localhost:8080 --catalog hive --schema acct10001 --user admin --debug --output-format=CSV_HEADER_UNQUOTED --execute "SELECT *, '10001' as ebs_account_id from aws_line_items where bill_billingentity = 'AWS Marketplace' and product_productname like '%Red Hat%';"`

You should see a bunch of rows, around 721
